torchok/data/datasets/detection/detection.py

```diff
@@ -169,11 +170,11 @@ def __getitem__(self, idx: int) -> dict:
 
     def collate_fn(self, batch):
         r"""Puts each data field into a tensor with outer dimension batch size"""
```

Comment:
I think you need to add torch issue in documentation to explain why we need tensors